addons/editor-theme3/theme3.js

```diff
@@ -105,5 +105,5 @@ export default async function ({ addon, global, console }) {
     }
   }
 
-  document.querySelector(".scratch-addons-theme[data-addon-id='editor-theme3']").textContent += stylesheet;
+  document.querySelector("[data-addon-id='editor-theme3']").textContent += stylesheet;
```

Comment:
Bug: after reenable, color for custom blocks is not restored

Reply:
This happens because the userscript modifies its own userstyle, and cs.js uses the text of the element to check if it's the same